docs/quickstarts/rosa-edit-app-ingress/rosa-edit-app-ingress.yml

```diff
@@ -0,0 +1,52 @@
+
+metadata:
+  name: rosa-edit-app-ingress
+  instructional: true
+spec:
+  displayName: Edit your ROSA cluster's application ingress
+  durationMinutes: 5
+  type:
+    text: Quick start
+    color: green
+  icon: data:image/svg+xml;base64,PCEtLSBHZW5lcmF0ZWQgYnkgSWNvTW9vbi5pbyAtLT4KPHN2ZyB2ZXJzaW9uPSIxLjEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgd2lkdGg9IjUxMiIgaGVpZ2h0PSI1MTIiIHZpZXdCb3g9IjAgMCA1MTIgNTEyIj4KPHRpdGxlPjwvdGl0bGU+CjxnIGlkPSJpY29tb29uLWlnbm9yZSI+CjwvZz4KPHBhdGggZD0iTTQ0OCA2NHY0MTZoLTMzNmMtMjYuNTEzIDAtNDgtMjEuNDktNDgtNDhzMjEuNDg3LTQ4IDQ4LTQ4aDMwNHYtMzg0aC0zMjBjLTM1LjE5OSAwLTY0IDI4LjgtNjQgNjR2Mzg0YzAgMzUuMiAyOC44MDEgNjQgNjQgNjRoMzg0di00NDhoLTMyeiI+PC9wYXRoPgo8cGF0aCBkPSJNMTEyLjAyOCA0MTZ2MGMtMC4wMDkgMC4wMDEtMC4wMTkgMC0wLjAyOCAwLTguODM2IDAtMTYgNy4xNjMtMTYgMTZzNy4xNjQgMTYgMTYgMTZjMC4wMDkgMCAwLjAxOS0wLjAwMSAwLjAyOC0wLjAwMXYwLjAwMWgzMDMuOTQ1di0zMmgtMzAzLjk0NXoiPjwvcGF0aD4KPC9zdmc+Cg==
+  prerequisites:
+    - You must have a ROSA cluster.
+    - You must be a cluster owner, cluster editor, or Organization Administrator for the cluster.
+
+  description: |-
+    Edit the application ingress of your ROSA cluster after creation.
+  introduction: |-
+    Using OpenShift Cluster Manager, you can edit the application ingress of your ROSA clusters. In this quick start, you'll learn how to edit your ROSA cluster's application ingress.
+
+  tasks:
+    - title: Edit application ingress for your cluster
+      description: |-
+        To edit your cluster's application ingress:
+
+        1. Go to **Cluster List**.
+        1. Click your cluster's name to view cluster details.
+        1. Click the **Networking** tab.
+        1. In the Application Ingress section, click the **Edit application ingress** button.
+
+            a. You can change the Default application router if needed.
+
+            b. Click the checkbox if you want to make your router private.
+
+        1. Click **Save** to finalize your changes.
+
+        Your cluster should now have the newly updated application ingress..
+
+  # optional - the task's Check your work module
+      review:
+        instructions: |-
+          - Does your cluster display the new application ingress you changed it to?
+        failedTaskHelp: Try following the steps again.
+  # optional - the task's success and failure messages
+      summary:
+        success: Shows a success message in the task header
+        failed: Shows a failed message in the task header
+  conclusion: |-
+        Congratulations, you've edited your cluster's application ingress!
+
+       You can edit your application ingress as often as needed.
```

Comment:
OK, This is the issue! This line has to be indented more to the right. Classic YAML